packages/storage-postgres/test/workflow-postgres-repo_test.ts

```diff
@@ -0,0 +1,152 @@
+import { PrismaClient } from '@prisma/client';
+import { v4 } from 'uuid';
+import { expect } from 'chai';
+import { WorkflowStateRepositoryPostgres } from '../src/repo/workflow-postgres-repo';
+
+describe('Workflow state postgres repo', () => {
+  const prisma = new PrismaClient({
+    datasourceUrl: process.env.DATABASE_URL,
+  });
+  const repo = new WorkflowStateRepositoryPostgres(prisma);
+
+  async function initialise() {
+    const params = {
+      workflowId: v4(),
```

Comment:
I think we should consider using `v7` https://www.npmjs.com/package/uuid#uuidv7options-buffer-offset

would create sequential uuids

Reply:
comment doesn't belong in the test, but was where I saw a v4